topics/activitypub/index.md

```diff
@@ -0,0 +1,11 @@
+---
+display_name: ActivityPub
+github_url: https://github.com/w3c/activitypub
+logo: activitypub.png
+related: decentralized, federated, protocol, social-network, w3c
+short_description: Decentralized social networking protocol.
```

Comment:
Can you rephrase to a full sentence? Maybe "ActivityPub is an open, decentralized social networking protocol."